lib/helpers.nix

```
@@ -51,6 +51,10 @@ with lib; rec {
     then "nil"
     else "";
 
+  mkUnkeyed = list:
```

```suggestion
  listToUnkeyedAttrs = list:
```
Let's have a very clear name.